File: Q7.c
int reverse(int x){
    long long rev=0;
    
    for(int i=x;i!=0;i=i/10) {
        rev=rev*10+i%10;
    }
    if(rev<-2147483648 || rev>2147483647) return 0;
    else return rev;
}

File: Q7.py
class Solution:
    def reverse(self, x: int) -> int:
        if x<0:
            a=int(str(abs(x))[::-1])
            if a<-2147483648 or a>2147483647:
                return 0
            else:
                return -a

        else: 
            a=int(str(x)[::-1])
            if a<-2147483648 or a>2147483647:
                return 0
            else:
                return a
        
